src/main/java/mate/academy/intro/model/Book.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.intro.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+
+@Entity

```

Comment:
Have you tested your solution? 
```suggestion
@Entity
@Getter
@Setter
```